### PR TITLE
Upgrade Backbone Forms; move overrides out

### PR DESCRIPTION
**Pull request by tomgreenfield**
Resolves https://github.com/adaptlearning/adapt_authoring/issues/1544.

### CODE REVIEW

**Comment by dancgray** [comment]
@tomgreenfield items display seems a little broken:

screen shot 2017-07-28 at 12 27 18

**Comment by taylortom** [comment]
Good spot @dancgray. We seem to be missing the classes from the item divs:

## Broken
screen shot 2017-07-28 at 13 50 15

## Working
screen shot 2017-07-28 at 13 52 05

File: frontend/src/libraries/backbone-forms-lists.js [hunk 1]
```diff
@@ -402,11 +397,10 @@ define(['jquery', 'underscore', 'backbone', 'backboneForms'], function($, _, Bac
   }, {
 
     //STATICS
-    // TODO remove this horrible code and externalise
     template: _.template('\
-      <div class="list-item clearfix">\
```
Comment (taylortom) [inline frontend/src/libraries/backbone-forms-lists.js]: Should this have moved to overrides?

**Comment by tomgreenfield** [comment]
Thanks Sherlock and Watson (@dancgray & @taylortom).

**Pull request merged**